src/display_methods.jl

```diff
@@ -77,10 +77,10 @@ function Base.display(report::Report, m::MIME"application/pdf", data)
     figname = add_figure(report, data, m, ".pdf")
 end
 
-#Text is written to stdout, called from "term" mode chunks
+# Text is written to stdout, called from "term" mode chunks
 function Base.display(report::Report, m::MIME"text/plain", data)
     io = PipeBuffer()
-    show(IOContext(io, :limit => true), m, data)
+    show(IOContext(io, :limit => true, :color => true), m, data)
```

Comment:
Are you sure the `:color => true` part is a good idea? Most applications don't handle ANSI escape codes very well.

Reply:
thanks for checking, I found there is no ANSI code processing within this package, so it should be removed.I think.

Rather, it turns out that we actually have escaped ANSI chars [here](https://github.com/JunoLab/Weave.jl/blob/avi/limitdisplay/src/run.jl#L271).